DOC/BuildVentoyFromSource.txt

```diff
@@ -1,30 +1,41 @@
 
 ==========================================
-1. Compile Enviroment
+1. Operating system
 ==========================================
     My build envrioment is CentOS 7.8 x86_64. So here I first explain how to create the build environment from scratch.
     Because Ventoy is based on many open source projects, so the environment is important. I suggest you test it on a virtual machine firstly.
 
 1.1 Install CentOS 7.8
     I use CentOS-7-x86_64-Everything-2003.iso and select Minimal install
     
-1.2 Install Packages    
-    yum install \
-        libXpm net-tools bzip2 wget vim gcc gcc-c++ samba dos2unix glibc-devel glibc.i686 glibc-devel.i686 \
-        mpfr.i686 mpfr-devel.i686 zlib.i686 rsync autogen autoconf automake libtool gettext* bison binutils \
-        flex device-mapper-devel SDL libpciaccess libusb freetype freetype-devel gnu-free-* qemu-* virt-* \
-        libvirt* vte* NetworkManager-bluetooth brlapi fuse-devel dejavu* gnu-efi* pesign shim \
-        iscsi-initiator-utils grub2-tools zip nasm acpica-tools glibc-static zlib-static xorriso
-    
-
-
 ==========================================
-2. Download Source Code
+2. Download Ventoy Source Code
 ==========================================
 2.1 Download Ventoy source code from github and decompress it. 
     Next I assume that you have unzipped the code into the /home directory (check /home/Ventoy-master/README.md file for the directory layout).
+
+==========================================
+3. Prepare the build environment
+==========================================
+3.1  Helper script:
+
+    You can skip steps 3.2, 3.3 and 3.4 with a helper script:
+    cd /home/Ventoy-master/INSTALL
+    sh prepare_build_environment.sh
+    
+    The script is tested to work with CentOS7 x86_64
     
-2.2 Download third-part source code and tool
+    If you use it, you can jump to the step 3.5 after the running the script
+
+3.2 Install required dependencies from the Operating System:
+    sudo yum install \
+         libXpm net-tools bzip2 wget vim gcc gcc-c++ samba dos2unix glibc-devel glibc.i686 glibc-devel.i686 \

```

Comment:
To be honest, when I tried to run this at the CentOS7 container, I got a conflict because zlib wanted to be installed for i686...